src/api/app/controllers/webui/project_controller.rb

```diff
@@ -14,15 +14,15 @@ class Webui::ProjectController < Webui::WebuiController
                                      :new_package, :new_package_branch, :incident_request_dialog, :release_request_dialog,
                                      :show, :linking_projects, :add_person, :add_group, :buildresult, :delete_dialog,
                                      :destroy, :remove_path_from_target, :rebuild_time, :packages_simple,
-                                     :requests, :save, :monitor, :toggle_watch, :meta,
+                                     :requests, :save, :monitor, :toggle_watch,
```

Comment:
This looks like a cleanup of the last commit. Please squash them together

Reply:
it was a required part of my changes, yes. I squashed now the commits.